ui/raidboss/raidemulator.js

```diff
@@ -41,7 +41,7 @@ import './raidemulator.css';
   let logConverterWorker;
 
   document.addEventListener('DOMContentLoaded', () => {
-    emulator = new RaidEmulator(Options);
+    emulator = new RaidEmulator();
```

Comment:
This change (and associated `RaidEmulator` class changes I feel aren't the best way to address this? It seems to me that it would make more sense to move the block of `new <some class>` constructor calls and associated `...on(` event listener binds down to be within the event handler for loading config instead, minus the `Persistor` constructor?

Otherwise this makes `RaidEmulator` not follow the general pattern of the rest of the code, where it accepts the options in the constructor.

Reply:
Yeah, agreed.  I think `RaidEmulator` already isn't following the pattern of the rest of the code to defer its constructions, so that seems like a good suggestion all around.

Reply:
Done.